php-cs-fixer

```diff
@@ -11,28 +13,34 @@
  * with this source code in the file LICENSE.
  */
 
+use Composer\XdebugHandler\XdebugHandler;
+use PhpCsFixer\Console\Application;
+
 error_reporting(E_ALL & ~E_DEPRECATED & ~E_USER_DEPRECATED);
 
-set_error_handler(static function ($severity, $message, $file, $line) {
+set_error_handler(static function (int $severity, string $message, string $file, int $line): bool {
+    // @phpstan-ignore if.condNotBoolean
     if ($severity & error_reporting()) {
         throw new ErrorException($message, 0, $severity, $file, $line);
     }
+
+    return true;
 });
 
 // check environment requirements
-(function () {
-    if (\PHP_VERSION_ID === 80000) {
+(static function (): void {
+    if (\PHP_VERSION_ID === 8_00_00) {
         fwrite(STDERR, "PHP CS Fixer is not able run on PHP 8.0.0 due to bug in PHP tokenizer (https://bugs.php.net/bug.php?id=80462).\n");
         fwrite(STDERR, "Update PHP version to unblock execution.\n");
 
         exit(1);
     }
 
-    if (\PHP_VERSION_ID < 70400 || \PHP_VERSION_ID >= 80400) {
+    if (\PHP_VERSION_ID < 7_04_00 || \PHP_VERSION_ID >= 8_04_00) {
```

Comment:
`\PHP_VERSION_ID < 7_04_00` is never true.

Reply:
@kubawerlos Why?

Reply:
- in PHP 7.4.0 and higher it is `true`
- in PHP lower than 7.4.0 ii is a syntax error

Reply:
Good point - I have rewritten the integer constants in this entry file using `(int) 'xxx'` syntax.

The minimal non-syntax-error PHP version was PHP 7.0.0. Newly, it is the same.

Reply:
@kubawerlos Thanks for clarification - I did not thought about numeric literal separator being not compatible 🤦. Remember: providing the reasoning directly minimises the risk of discussion ping-pong 😆.

Reply:
> providing the reasoning directly minimises the risk of discussion ping-pong 😆.

It also makes it impossible to find out yourself

Reply:
> It also makes it impossible to find out yourself

Is code review actually the place for "finding out by yourself"? I am not so sure. We all have limited time, I prefer to get explicit information about the reasoning rather than figuring it out. The clearer the comments are, the less confusion and needless discussions they produce.

Reply:
Let's merge the PR, all discussion threads have been resolved.